API/hermes/hermes.cpp

```diff
@@ -2623,6 +2641,10 @@ void HermesRuntimeImpl::throwJSErrorWithMessage(Args &&...args) {
   throwPendingError();
 }
 
+void HermesRuntimeImpl::createNodeApiEnv(int32_t apiVersion, napi_env *env) {
+  ::hermes::node_api::createNodeApiEnv(*this->getVMRuntimeUnsafe(), env);
```

Comment:
I might want to wrap this in a try-catch to ensure only `JSINativeException` are thrown? 🤔